core/src/test/java/com/google/errorprone/bugpatterns/UnusedVariableTest.java

```diff
@@ -1385,7 +1385,7 @@ public void simpleRecord() {
     assumeTrue(RuntimeVersion.isAtLeast16());
     helper
         .addSourceLines(
-            "SimpleRecord.java", //
+            "SimpleRecord.java",
```

Comment:
Please revert, the line comment forces google-java-format to add a line break

Reply:
Ah sorry. Do you think it would make sense to change the comment to make this clearer (for example by changing it to `// (force line break for google-java-format)`, in case that still works then)? Or is this not worth it?

Reply:
I'd prefer just `//`

(And hopefully in the not too distant future we can start using raw string literals, at least in tests...)